pandas/tests/conftest.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+import pandas.util._test_decorators as td
+
+
+@pytest.fixture(params=[None, 'gzip', 'bz2',
```

Comment:
@reidy-p yeah move this 1 level high in next PR (it’s already there just add the contents)